### ISSUE TITLE
KryptonStatusStrip - Canary - Build Error

### ISSUE THREAD

**Issue by gunny2k7** [Krypton-Suite/Standard-Toolkit#44]
Hi in Krypton Toolkit Stable the StatusStrip will not add to my form it errors so i tried a black project with Canary Build it adds it which is cool but when i press build and run the exe with 6.0.2103.4 it gives debug error for Me.KryptonStatusStrip1 = New Krypton.Toolkit.KryptonStatusStrip()  -  System.NullReferenceException: 'Object reference not set to an instance of an object.'

**Comment by Wagnerp** [comment]
Hi @gunny2k7 

Thanks for the report. I've fixed the bug in the upcoming 2104 build of the extended toolkit, and will be removing the control from the standard toolkit. I was trying to render custom progress bars, which don't exist on a newly created status strip #5

**Comment by lorinkundert** [comment]
For me the status strip works on a normal window but not on a KryptonForm, it will disappear and it is impossible to add it again. The error is the same as it has always been.

**Comment by Wagnerp** [comment]
Hi @lorinkundert 

Is that in the extended toolkit?

**Comment by lorinkundert** [comment]
Yes it is, I wonder if it could be a Net Core issue with the development environment in Visual studio.

**Comment by Smurf-IV** [comment]
> 
> 
> Yes it is, I wonder if it could be a Net Core issue with the development environment in Visual studio.

So, this should be transferred to the other project then ?!

**Comment by Wagnerp** [comment]
> Yes it is, I wonder if it could be a Net Core issue with the development environment in Visual studio.

I'm actually in the middle of redeveloping the `Extended-Toolkit`, so I'll have a look.
